bxml/buildingBlocks/forwardCall.md

```diff
@@ -0,0 +1,27 @@
+# Forward a Call
+
+## Concept
+
+To forward a call to another number, we will use the transfer BXML verb.
+
+## Use Cases
+
+| Use Case                                    | BXML Code                                                 |
+|:--------------------------------------------|:----------------------------------------------------------|
+| Call centers: A customer can call in and a recording will play. The call will then be transfered to the first available tenant. | Add multiple phone numbers(`<PhoneNumber>+15552221234</PhoneNumber>`) |
+| Appointment reminders: If a customer would like to reschedule, you can transfer their call to the scheduling office | the code will be the same as the example|
```

Comment:
`the code will be the same as the example`

Capital `T`

`The code will be the same as the example`